{"name": "User ID does not exist", "status": "passed", "steps": [{"name": "Given the DELETE header \"app-id\" is \"665acdd5b842089f2e37b051\"", "status": "passed", "start": 1717322784007, "stop": 1717322784007}, {"name": "And the DELETE request user ID is \"60d0fe4f5311236168a109f1\"", "status": "passed", "start": 1717322784007, "stop": 1717322784007}, {"name": "When I make a DELETE request to it", "status": "passed", "start": 1717322784007, "stop": 1717322784330}, {"name": "Then the DELETE status code should be 404", "status": "passed", "start": 1717322784330, "stop": 1717322784330}, {"name": "And the DELETE response should contain \"error\" with value \"RESOURCE_NOT_FOUND\"", "status": "passed", "start": 1717322784330, "stop": 1717322784331}], "start": 1717322784006, "stop": 1717322784331, "uuid": "11738275-5bcc-40c7-a64b-6c4bfd984679", "historyId": "14137fc192ebb1ad0dc53dbfdb3c532b", "fullName": "Api testing for delete endpoint: User ID does not exist", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Api testing for delete endpoint"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}